style/battle.css

```diff
@@ -630,7 +630,7 @@ License: GPLv2
 	background: #333333;
 	color: #CCCCCC;
 }
-.dark .battle-log .chat strong,
+.dark .battle-log .chat > strong,
```

Comment:
I meant that I'm unhappy with CSS not making a good solution possible to this problem, not with your CSS which makes the best of a bad situation.